cni/pkg/nodeagent/ztunnelserver.go

```diff
@@ -327,6 +327,7 @@ func (z *ztunnelServer) PodAdded(ctx context.Context, pod *v1.Pod, netns Netns)
 	if err != nil {
 		return err
 	}
+	log.Debug("sent pod add to ztunnel")
```

Comment:
nit: if we keep this, make it a `trace`, we already have quite a bit of debug logging in the node agent.

Reply:
unfortunately there is no trace in our go logging

Reply:
I mostly added this since it wasn't immediately clear in the logs that the request successfully sent. I'm fine with removing this.

Reply:
should we add pod name here to make it more meaningful?

Reply:
The pod name is already included in the log here https://github.com/istio/istio/pull/53888/files#diff-9873062ab6cdec1c9e1829c5cad625c4ef9d878a05b8b96ee3449623ea50f442R312